src/test/java/api/support/fixtures/InstanceExamples.java

```diff
@@ -53,4 +53,119 @@ static InstanceBuilder basedUponDunkirk(
     return new InstanceBuilder("Dunkirk", booksInstanceTypeId)
       .withContributor("Nolan, Christopher", personalContributorNameTypeId);
   }
+  
+  static InstanceBuilder basedUponOneOfHundredBookInstances(
```

Comment:
Personally, I might have split the test for this kind of scenario into a separate class and kept the specific support 
 like this with that code, rather than including it in the examples and fixtures straight away.

Reply:
Sure. Will update.